client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -225,6 +225,7 @@ export const JurisdictionAdminView: React.FC = () => {
           auditBoards={auditBoards}
           auditType={auditSettings.auditType}
           auditName={auditSettings.auditName}
+          isAuditOnline={!!auditSettings.online}
```

Comment:
`auditSettings.online` should already be a boolean, so i don't think the `!!` is necessary. also at this point i feel like we might as well pass in the whole `auditSettings` object, though this way is fine too

Reply:
@jonahkagan it's actually `boolean | null` which is why I went this route.